packages/carbon-graphs/tests/unit/controls/Bubble/SingleDataset/BubbleSingleDatasetPanning-spec.js

```diff
@@ -351,5 +351,26 @@ describe('Bubble Single Dataset - Panning', () => {
         expect(legendItem.getAttribute('aria-current')).toBe('true');
       });
     });
+    describe('when same data is passed on multiple clicks of panning', () => {
```

Comment:
nit: when *the* same data

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/66/commits/1f556c88eae25ac272b14101b4dd705321f965ad#diff-10d5d54c3e7d7f730a2bdd1711f7dc6605e5d787e88735b3d019c468551026a4R354).